pkg/authorization/local_authorizer.go

```diff
@@ -39,16 +40,16 @@ const (
 )
 
 type LocalAuthorizer struct {
-	roleLister               rbaclisters.RoleLister
-	roleBindingLister        rbaclisters.RoleBindingLister
-	clusterRoleBindingLister rbaclisters.ClusterRoleBindingLister
-	clusterRoleLister        rbaclisters.ClusterRoleLister
+	roleLister               rbacv1listers.RoleClusterLister
+	roleBindingLister        rbacv1listers.RoleBindingClusterLister
+	clusterRoleBindingLister rbacv1listers.ClusterRoleBindingClusterLister
+	clusterRoleLister        rbacv1listers.ClusterRoleClusterLister
 
 	// TODO: this will go away when scoping lands. Then we only have those 4 listers above.
```

Comment:
Are we able to collapse this now?

Reply:
oops, missed that - added a commit for it